### PR TITLE
Import maps

### PR DESCRIPTION
**Pull request by internettrans**

### CODE REVIEW

File: text/0004-import-maps.md
Comment (dkayiwa) [inline text/0004-import-maps.md:39]: Any reason for making it different from the specification?

Comment (internettrans) [reply text/0004-import-maps.md:39]: The only difference is changing the `<script type="importmap">` to `<script type="systemjs-importmap">`. The inner json is exactly the same.

I believe the reason that it is different is so that the developer has full control over whether they are using native browser import maps vs systemjs import maps. Using the exact same script type could result in users accidentally using native browser import maps instead of SystemJS if they update their browser to a version that uses native import maps. Since the specification for import maps is still slightly evolving, it is safer to stick to SystemJS until we're confident that our code will work in all browsers' implementation of import maps.

^^ This is my best guess as to the explanation, but if you want to ask the author of SystemJS, you can open an issue in https://github.com/systemjs/systemjs asking about it. I am a contributor to the project, but not the primary author and maintainer.

Comment (dkayiwa) [reply text/0004-import-maps.md:39]: Oh i see! Got you. 😊

**Review by fatmali — APPROVED** [review]

**Review by brandones — APPROVED** [review]
LGTM

**Comment by internettrans** [comment]
Need one more approval, either from @angshu or @ojwanganto

**Comment by angshu** [comment]
LGTM

**Pull request merged**
